src/app/screen/home/home.component.scss

```
@@ -1,24 +1,21 @@
 .home-container {
-  width: 100%;
-  height: 2000px;
-  background-color: gray;
-  display: flex;
-
-  .right-contents {
-    width: 15%;
+  width: 375px;
+  height: 812px;
+  padding: 40px 28px;
+  box-sizing: border-box;
+  background-color: #F0E5D0;
+  .underline {
```

可読性向上のため、セレクタ追加の際に改行を入れましょう
```suggestion

  .underline {
```